humble/distribution.yaml

```diff
@@ -4036,11 +4036,15 @@ repositories:
       version: master
     status: maintained
   libfranka:
+    doc:
+      type: git
+      url: https://github.com/frankaemika/libfranka-release.git
```

Comment:
Often the doc entry [is the source repo](https://docs.ros.org/en/rolling/How-To-Guides/Documenting-a-ROS-2-Package.html#ci-docs-ros-org) rather than the release repo so that documentation can be updated without making a new release. Are you sure you want the documentation to be built from the release repo?

Reply:
Yes true -> redirected.

Other question, that now the version changed is not a problem for the existing releases right? Both the 0.13.6 and 0.14.2 will be available...?

Reply:
No, we only provide the latest version of the package.  Old versions are removed during the sync process.